### ISSUE TITLE
Module build fails

### ISSUE THREAD

**Issue by miranda-elliott** [qlik-demo-team/qdt-vue-template#1]
Module won't build. Tried re-installing vue-loader and updating vue-template-compiler. How can this be resolved? Thank you.

```
Module build failed: Error:

Vue packages version mismatch:

- vue@2.5.2
- vue-template-compiler@2.5.17

This may cause things to work incorrectly. Make sure to use the same version for both.
If you are using vue-loader@>=10.0, simply update vue-template-compiler.
If you are using vue-loader@<10.0 or vueify, re-installing vue-loader/vueify should bump vue-template-compiler to the l
test.

```

**Comment by jdriesen** [comment]
In case smb is interested ... 
I succeeded in modifying the package.json file....
Right now, I've everything working like a charm ...

**Comment by yianni-ververis** [comment]
Than kyou johny. Can you commit the package.json?

**Comment by jdriesen** [comment]
mmm ... my App was running last week, now not anymore ...
Gonna check what happened.

Coming back on this asap.

**Comment by jdriesen** [comment]
If I can ask ... did smt change on DATA side ?
As soon as I click on a component, I get the spinner ... 
(and this was not the case last week...)

Thanks in advance for your reply,
Johnny

**Comment by yianni-ververis** [comment]
You have updated your packages so I do not know about any conflicts.

This is working fine,
https://webapps.qlik.com/qdt-components/vue/index.html#/

**Comment by jdriesen** [comment]
I agree ...
I updated the packages ... 
Though, the data is taken from 
const options = {
  config: {
    host: 'sense-demo.qlik.com',
    secure: true,
    port: 443,
    prefix: '',
    appId: '133dab5d-8f56-4d40-b3e0-a6b401391bde',
  },
  connections: {
    vizApi: true,
    engineApi: true,
  },
};

(this is set in file QdtComponent.vue)

So, I thought maybe smt is changed at data side ...

Grtz,
Johnny

**Comment by jdriesen** [comment]
I think I found the cause ... 
(but I don't know what to do about it) 

As far as I remember, this was not the case last week ....

![error](https://user-images.githubusercontent.com/1137648/44271925-b4822b80-a23b-11e8-9c5d-9b21775d751d.PNG)

**Comment by yianni-ververis** [comment]
close all webpack running services. The port you are using is not in the VP Whitelist.

You should point to your Qlik Server where you manage the Virtual Proxy.

**Comment by miranda-elliott** [comment]
Fixed by running `npm install vue@2.5.17`. The latest vue-template-compiler version is 2.5.17 so needed to downgrade from vue 2.5.2.
